### DISCUSSION TITLE
What is the filetype for Dressing's Telescope `vim.ui.select`?

### DISCUSSION THREAD

**Discussion by ziontee113** [stevearc/dressing.nvim#96] [Q&A] [answered]
Hi, I'm using this config for `nvim-colorizer`

```lua
return {
    "NvChad/nvim-colorizer.lua",
    config = {
        filetypes = {
            "*", -- Highlight all files, but customize some others.
            typescriptreact = { mode = "background", tailwind = true },
            svelte = { mode = "background", tailwind = true },
            rust = { mode = "foreground", tailwind = true },
        },
        user_default_options = {
            always_update = true,
        },
    },
}
```

I'm looking for a way to only have the `always_update` option when I search for HTML colors using `vim.ui.select`, so I need the filetype for Dressing's Telescope `vim.ui.select`. Thank you very much!

**Comment by stevearc** [comment, marked as answer]
You can find it yourself by running `:set filetype?`
```
filetype=DressingSelect
```

**Reply by ziontee113** [reply]
Thank you @stevearc for the answer. I was unable to perform that action because I don't know how to focus on that window. If I click into it, it will just disappear.
